pkg/minikube/cluster/cluster.go

```diff
@@ -192,6 +194,8 @@ type MachineConfig struct {
 func engineOptions(config MachineConfig) *engine.Options {
 	o := engine.Options{
 		Env:              config.DockerEnv,
+		AddRegistry:      config.AddRegistry,
+		BlockRegistry:    config.BlockRegistry,
```

Comment:
@praveenkumar As I said, that's probably because this is not "Docker's Inc Docker (now moby)", but "[Red Hat's docker](http://www.projectatomic.io/docs/docker_patches/)"